tests/articulation_tests.ts

```diff
@@ -256,7 +253,7 @@ const ArticulationTests = {
     // Helper function to justify and draw a 4/4 voice
     Formatter.FormatAndDraw(ctx, staveBar3, notesBar3);
     // bar 4 - juxtaposing second bar next to first bar
-    const staveBar4 = new Stave(staveBar3.width + staveBar3.x, staveBar3.y, 150);
+    const staveBar4 = new Stave(staveBar3.getWidth() + staveBar3.getX(), staveBar3.getY(), 150);
```

Comment:
Thanks for cleaning up these calls. Curiously did you notice any effect on test performance?

Reply:
I didn't notice anything regarding performance, especially since these getters are only called a few times.

We would need a larger test, something like a multi-page Bach Demo that uses a lot of getters, to see if it has a noticeable effect on performance.

This discussion does remind me: **Do we always need accessors for simple fields like x, y, width, height?** 

Sometimes it feels excessive, and it might reduce readability in areas of code where we do lots of calculations.

For example, `BoundingBox` has 4 protected members (x, y, w, h). So external code has to use .getX(), .getY(), .getW(), .getH(). BoundingBox is just a entity for doing geometric calculations, so maybe x, y, w, h should all just be public? We can explore this in a separate PR if you want.

Reply:
It's a good point. I typically default to not using accessors for simple fields. But VexFlow is so big now, it's hard to draw the line around what _simple_ means, so we end up with a lot of inconsistency, which raises the cognitive load ("should this be "note.foo" or "note.getFoo()"). So I think it's easier on us to default to using accessors for everything.